src/kaleidoscope/driver/attiny/Base.h

```diff
@@ -0,0 +1,61 @@
+/* -*- mode: c++ -*-
+ * kaleidoscope::driver::attiny::base -- Base ATTiny driver
+ * Copyright (C) 2019  Keyboard.io, Inc
+ * Copyright (C) 2019  Dygma Lab S.L.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+namespace kaleidoscope {
+namespace driver {
+namespace attiny {
+
+struct BaseProps {
+  static constexpr uint8_t page_size = 64;
+  static constexpr uint8_t frame_size = 16;
+  static constexpr uint8_t blank = 0xff;
+  static constexpr uint8_t pages = 0;
+  static constexpr uint16_t firmware_length = 0;
+  static constexpr uint8_t delay = 1;
+
+  static struct {
+    static constexpr uint8_t page_address = 0x01;
+    static constexpr uint8_t continue_page = 0x02;
+    static constexpr uint8_t execute = 0x03;
+    static constexpr uint8_t erase_program = 0x04;
+    static constexpr uint8_t get_version_and_crc = 0x06;
+  } command;
```

Comment:
The commands appear to be the same for the Raise and the Model01, which is why I lifted them to the base class. Them being in a struct makes it impossible to override just one, mind you. But the struct looks nicer, and I expect that we won't need overrides for command ids anytime soon.